lnwire/onion_error_test.go

```diff
@@ -180,7 +180,7 @@ func TestFailUnknownPaymentHashOptionalAmount(t *testing.T) {
 	onionError := &FailUnknownPaymentHash{}
 
 	var b bytes.Buffer
-	if err := EncodeFailure(&b, onionError, 0); err != nil {
+	if err := EncodeFailure(&b, &onionError, 0); err != nil {
```

Comment:
commit message indicates that there were two cases, but i only see one. was the other fixed somewhere else?

Reply:
This commit has already been merged as part of #3188. What is left here is actually incorrect. Commit dropped.